### PR TITLE
Maia Metis Uni V3 fork volume

### PR DESCRIPTION
**Pull request by 0xLightt**

### CODE REVIEW

**Comment by llamatester** [comment]
The maia-v3 adapter exports: 
        
 
 _______________________________________
    Dexs for 2/4/2023
    _______________________________________
    
    
    
    Done in 4.24s.

**Comment by llamatester** [comment]
The maia-v3 adapter exports: 
        
 
 _______________________________________
    Dexs for 2/4/2023
    _______________________________________
    
    
    
    Done in 4.17s.

File: dexs/hermes-protocol/index.ts [hunk 1]
```diff
@@ -168,6 +168,11 @@ const adapter: SimpleAdapter = {
     [CHAIN.METIS]: {
       fetch,
       start: async () => 1670544000,
+      meta: {
```
Comment (dtmkeng) [inline dexs/hermes-protocol/index.ts]: could you undo this ?

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: dexs/maia-v3/index.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  ProtocolRevenue: "Protocol receives 10% of fees.",
+  SupplySideRevenue: "90% of user fees are distributed among LPs.",
+  HoldersRevenue: "Holders have no revenue."
+}
+
+const adapter: SimpleAdapter = {
+  adapter: {
+    [CHAIN.METIS]: {
+      fetch: v3Graphs(CHAIN.METIS),
+      start: getStartTimestamp({
```
Comment (dtmkeng) [inline dexs/maia-v3/index.ts]: can set ` start: async () => 1680307200`

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: dexs/hermes-protocol/index.ts (hunk 1 above)
Comment (0xLightt) [reply dexs/hermes-protocol/index.ts]: done

**Comment by llamatester** [comment]
The maia-v3 adapter exports: 
        
 
 _______________________________________
    Dexs for 2/4/2023
    _______________________________________
    
    
    
    Done in 3.88s.

File: dexs/maia-v3/index.ts (hunk 2 above)
Comment (0xLightt) [reply dexs/maia-v3/index.ts]: also done

**Comment by llamatester** [comment]
The maia-v3 adapter exports: 
        
 
 _______________________________________
    Dexs for 2/4/2023
    _______________________________________
    
    METIS 👇
    Backfill start time: 1/4/2023
    Timestamp: 1680479998
    Block: 5230762
    Total volume: 5862.222083117573
    Daily volume: 5862.222083117572313544787757257036
    Daily fees: 58.2632838769539834375623028915521
    Total fees: 74.6482756684377
    Daily protocol revenue: 5.82632838769539834375623028915521
    └─ Methodology: Protocol receives 10% of fees.
    Total protocol revenue: 7.46482756684377
    └─ Methodology: Protocol receives 10% of fees.
    Daily holders revenue: 0
    └─ Methodology: Holders have no revenue.
    Total holders revenue: 0
    └─ Methodology: Holders have no revenue.
    Daily user fees: 58.2632838769539834375623028915521
    └─ Methodology: User pays 0.01%, 0.05%, 0.30%, or 1% on each swap.
    Total user fees: 74.6482756684377
    └─ Methodology: User pays 0.01%, 0.05%, 0.30%, or 1% on each swap.
    Daily supply side revenue: 52.43695548925858509380607260239689
    └─ Methodology: 90% of user fees are distributed among LPs.
    Total supply side revenue: 67.18344810159393
    └─ Methodology: 90% of user fees are distributed among LPs.
    Daily revenue: 0
    Total revenue: 0
    
    
    
    
    Done in 6.92s.

**Pull request merged**